x-pack/plugins/enterprise_search/public/applications/analytics/components/analytics_collection_view/analytics_collection_integrate/analytics_collection_integrate_searchui.tsx

```diff
@@ -124,6 +124,16 @@ plugins: [
 ...
 }`}
           </EuiCodeBlock>
+          <EuiSpacer size="m" />
+          <p>
+            {i18n.translate(
+              'xpack.enterpriseSearch.analytics.collections.collectionsView.integrateTab.searchui.stepTwo.moreInfoDescription',
+              {
+                defaultMessage:
+                  'See the Javascript Embed and Javascript Client tabs for more information on initializing the tracker and firing events.',
```

Comment:
Here I refer to the use of `createTracker`, `trackEvent` and `trackPageView` functions, which depends on the chosen integration solution (JS Embed vs Client).

Reply:
Lets link them to the plugin page on search-ui docs on analytics plugin https://docs.elastic.co/search-ui/api/core/plugins/analytics-plugin if they want more concrete examples.

Reply:
Done. Please review the EUI structure of this section (`FormattedMessage` -> `EuiLink`), I want to make sure the components are wrapped correctly.

<img width="1045" alt="Screen Shot 2023-01-11 at 9 28 43 AM" src="https://user-images.githubusercontent.com/14224983/211818721-307ac277-da61-4f04-8093-6e99ed95cd07.png">